bin/floss.js

```diff
@@ -28,6 +28,8 @@ function parseArgs(args) {
     commander.option('-d, --debug', 'Launch electron in debug mode')
         .option('-p, --path [path/to/folder/or/file.js]', 'Either a path to a directory containing index.js or a path to a single test file')
         .option('-e, --electron [path/to/Electron]', 'Path to version of Electron to test on')
+        .option('-r, --reporter [spec]', 'Mocha reporter for headless mode only')
+        .option('-r, --reporter-options [spec]', 'Additional arguments for reporter options')
```

Comment:
would be good to specify expected as a querystring and dependent on the reporter specified.  would this option be valid without the -r option specified?  if not maybe we could say it's ignored unless the -r option is set?